File: single-language-analysis/dataset-files/python/test_smell_141.py
# Original URL: https://github.com/ibmdbanalytics/ibmdbpy/blob/c7f1cd3892b609644e3d68fcbdec151ac4634612/ibmdbpy/tests/test_frame.py#L296-L299

class Test_DataExploration(object):

    def test_smell_141(self, idadb, idadf, df):
        ida_head = idadf.head(10)
        assert isinstance(ida_head, pandas.core.frame.DataFrame)
        assert len(ida_head) == 10

File: single-language-analysis/dataset-files/java/TestSmell141.java
import org.junit.Test;
import static org.junit.Assert.*;
import pandas.core.frame.DataFrame;

public class TestSmell141 {

    @Test
    public void testIdadfHead10() {
        DataFrame idaHead = idadf.head(10);
        assertTrue(idaHead instanceof DataFrame);
        assertEquals(10, idaHead.length());
    }
}